test/sharedSteps.ts

```diff
@@ -187,10 +197,28 @@ export async function createSpellAndAddToDeal(
   await fluenceConfig.$commit();
 }
 
-export async function waitUntilShowSubnetReturnsSpell(
+function sortSubnetResult(result: WorkerServices) {
+  return result
+    .sort((a, b) => {
+      if (a.host_id < b.host_id) {
+        return -1;
+      }
+
+      if (a.host_id > b.host_id) {
+        return 1;
+      }
+
+      return 0;
+    })
```

Comment:
I suggest to use lodash sort here.

Reply:
`sortBy(result, 'host_id')`

Reply:
The bad thing about lodash is that id doesn't care about type information so you might run into the problem because of that